ci-tasks

```diff
@@ -6,7 +6,9 @@ set -e -x
 rake package
 
 # run the osc source validator to check the .spec and .changes locally
-(cd package && /usr/lib/obs/service/source_validator)
+# Workaround for CI failure:
+# Don't validate on SLE-15-SP3 since that image has an outdated validator
```

Comment:
I'd like to understand the problem before disabling the check. Any pointers?

Reply:
It had complained "you can have either multibuild or multiple .spec files, but not both". @joseivanlopez knows more.

Reply:
In the SP3 builds we use Leap 15.3 based image which contains an old source validator which does not properly support multibuild packages.

In `master` (TW based image) with a newer version it works properly.